src/patternfly/components/Page/page.scss

```diff
@@ -287,6 +296,37 @@
     }
   }
 
+  .pf-c-button {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--pf-c-page__header-tools-group--m-icons--c-button--Width);
+    height: var(--pf-c-page__header-tools-group--m-icons--c-button--Height);
+    padding: unset;
+    background-color: var(--pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor);
+    border-radius: var(--pf-c-page__header-tools-group--m-icons--c-button--BorderRadius);
+
+    &:active,
+    &.pf-m-active {
+      --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: var(--pf-c-page__header-tools-group--m-icons--c-button--active--BackgroundColor);
+    }
+
+    &:focus,
+    &.pf-m-focus {
+      --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: var(--pf-c-page__header-tools-group--m-icons--c-button--focus--BackgroundColor);
+
+      /* stylelint-disable */
+      .pf-c-notification-badge {
```

Comment:
looks like this style should also apply to the `:active` state, since this is what it looks like now

<img width="67" alt="Screen Shot 2020-01-10 at 12 09 12 PM" src="https://user-images.githubusercontent.com/35148959/72175601-1cda2b80-33a2-11ea-835f-2fdc707efade.png">

Also we should move `:active` after `:focus` since they modify the same property and an active element is also focused, otherwise `:active` styles will will always be overwritten by the `:focus` styles.

Reply:
sounds good, updated